examples/md-flexible/scripts/README.md

```diff
@@ -20,6 +20,29 @@ Requirements:
 
 Creates an interactive plot for each md-flex output file given. If no input is given it tries to find the last output folder of testTuning.py and plot everything in there.
 
+### plotPrediction.py
+
+Requirements:
+* python3 (tested with Python 3.8.3rc1)
+* [plotly](https://github.com/plotly/plotly.py) (tested with 4.7.1)
+
+Creates either a plot of the predictions or of the prediction and the tested times for each configuration for each md-flex output file given ddepending on the chosen option in the input.
+
+### plotDiffPredictionTest.py
+
+Requirements:
+* python3 (tested with Python 3.8.3rc1)
+* [plotly](https://github.com/plotly/plotly.py) (tested with 4.7.1)
+
+Creates a plot either of the total or the relative difference between the predictions and the tested times for each configuration for each md-flex output file given or both depending on the chosen option in the input.
+
+### compareOptimumConfiguration.py
+
+Requirements:
+* python3 (tested with Python 3.8.3rc1)
+
+Compares the selected optimum configuration after each tuning phase with a base file and every other file given in the input and prints the percentage of the alignment.
```

Comment:
```suggestion
Compares the tuning behavior of multiple runs of md-flexible. Specifically it compares the configuration which is selected as optimum after each tuning phase with a base file and every other file given in the input and prints the percentage of the alignment.
```
Question: If fileA contains one tuning phase and fileB contains two against what (if at all) is the second tuning phase of file2 compared? Clarify this in either this description or the help message.